plugins/Wallet/js/components/newwalletdialog.js

```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react';
+
+const NewWalletDialog = ({visible, password, seed, actions}) => {
+	if (!visible) {
+		return (
+			<div></div>
```

Comment:
not sure I like this pattern. Seems like the parent should be the one deciding whether or not to render the child, rather than always rendering a sometimes-invisible component.